tests/functional/aws-node-sdk/test/object/deleteObject.js

```diff
@@ -13,70 +20,225 @@ describe('DELETE object', () => {
         const s3 = bucketUtil.s3;
         const testfile = Buffer.alloc(1024 * 1024 * 54, 0);
 
-        before(() => {
-            process.stdout.write('creating bucket\n');
-            return s3.createBucketPromise({ Bucket: bucketName })
-            .then(() => {
-                process.stdout.write('initiating multipart upload\n');
-                return s3.createMultipartUploadPromise({ Bucket: bucketName,
-                    Key: objectName });
-            })
-            .then(res => {
-                process.stdout.write('uploading parts\n');
-                uploadId = res.UploadId;
-                const uploads = [];
-                for (let i = 1; i <= 3; i++) {
-                    uploads.push(
-                        s3.uploadPartPromise({ Bucket: bucketName,
-                            Key: objectName, PartNumber: i, Body: testfile,
-                            UploadId: uploadId })
-                    );
-                }
-                return Promise.all(uploads);
-            })
-            .catch(err => {
-                process.stdout.write(`Error with uploadPart ${err}\n`);
-                throw err;
-            })
-            .then(res => {
-                process.stdout.write('about to complete multipart upload\n');
-                return s3.completeMultipartUploadPromise({
+        describe('with multipart upload', () => {
+            before(() => {
+                process.stdout.write('creating bucket\n');
+                return s3.createBucketPromise({ Bucket: bucketName })
+                .then(() => {
+                    process.stdout.write('initiating multipart upload\n');
+                    return s3.createMultipartUploadPromise({ Bucket: bucketName,
+                        Key: objectName });
+                })
+                .then(res => {
+                    process.stdout.write('uploading parts\n');
+                    uploadId = res.UploadId;
+                    const uploads = [];
+                    for (let i = 1; i <= 3; i++) {
+                        uploads.push(
+                            s3.uploadPartPromise({ Bucket: bucketName,
+                                Key: objectName, PartNumber: i, Body: testfile,
+                                UploadId: uploadId })
```

Comment:
I would format this like so:
```suggestion
                            s3.uploadPartPromise({ 
                                Bucket: bucketName,
                                Key: objectName, 
                                PartNumber: i, 
                                Body: testfile,
                                UploadId: uploadId
                            })
```